README.md

```diff
@@ -31,21 +31,27 @@ A Terminal Emulator based on UWP and web technologies.
 - Improved tabs
 - Split screen support
 
-## How to install
+## How to install (as an end-user)
 
+- Download the latest [release](https://github.com/felixse/FluentTerminal/releases)
 - [Enable sideloading apps](https://www.windowscentral.com/how-enable-windows-10-sideload-apps-outside-store)
   - Alternatively, [enable developer mode](https://docs.microsoft.com/en-US/windows/uwp/get-started/enable-your-device-for-development) if you plan to do UWP app development. **For most users that do not intend to do UWP app development, developer mode will not be necessary.**
 - Install the *.cer file into `Local Machine` -> `Trusted Root Certification Authorities`
+  - This will require administrator elevation. If you installed the certificate and did not have to elevate, you likely installed it into an incorrect certificate store.
```

Comment:
maybe add something like 'do not trust the doubleclick'? 😄

Reply:
Oh, yeah, good one. I'll add something specific. Think we need screenshots?

Reply:
I added a blurb that automated installers should be avoided to ensure they get a chance to choose all of the correct installation options.